ntbs-service/DataAccess/NtbsContext.cs

```diff
@@ -560,19 +558,18 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 entity.Property(e => e.EmailSecondary).HasMaxLength(50);
                 entity.Property(e => e.Notes).HasMaxLength(500);
 
-                entity.HasKey(e => e.Username);
```

Comment:
I think we should still leave an index on `Username`, and make sure that it's unique.